apis/namespaced/apigee/v1beta1/zz_keystoresaliaseskeycertfile_types.go

```diff
@@ -0,0 +1,225 @@
+
+	ValidFrom *string `json:"validFrom,omitempty" tf:"valid_from,omitempty"`
+
+	Version *float64 `json:"version,omitempty" tf:"version,omitempty"`
+}
+
+type CertInfoParameters struct {
+
+	// +kubebuilder:validation:Optional
+	BasicConstraints *string `json:"basicConstraints,omitempty" tf:"basic_constraints,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	ExpiryDate *string `json:"expiryDate,omitempty" tf:"expiry_date,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	IsValid *string `json:"isValid,omitempty" tf:"is_valid,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	Issuer *string `json:"issuer,omitempty" tf:"issuer,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	PublicKey *string `json:"publicKey,omitempty" tf:"public_key,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	SerialNumber *string `json:"serialNumber,omitempty" tf:"serial_number,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	SigAlgName *string `json:"sigAlgName,omitempty" tf:"sig_alg_name,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	Subject *string `json:"subject,omitempty" tf:"subject,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	SubjectAlternativeNames []*string `json:"subjectAlternativeNames,omitempty" tf:"subject_alternative_names,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	ValidFrom *string `json:"validFrom,omitempty" tf:"valid_from,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	Version *float64 `json:"version,omitempty" tf:"version,omitempty"`
+}
+
+type CertsInfoInitParameters struct {
+	CertInfo []CertInfoInitParameters `json:"certInfo,omitempty" tf:"cert_info,omitempty"`
+}
+
+type CertsInfoObservation struct {
+	CertInfo []CertInfoObservation `json:"certInfo,omitempty" tf:"cert_info,omitempty"`
+}
+
+type CertsInfoParameters struct {
+
+	// +kubebuilder:validation:Optional
+	CertInfo []CertInfoParameters `json:"certInfo,omitempty" tf:"cert_info,omitempty"`
+}
+
+type KeystoresAliasesKeyCertFileInitParameters struct {
+	Cert *string `json:"cert,omitempty" tf:"cert,omitempty"`
+
+	CertsInfo *CertsInfoInitParameters `json:"certsInfo,omitempty" tf:"certs_info,omitempty"`
+
+	KeySecretRef *v1.LocalSecretKeySelector `json:"keySecretRef,omitempty" tf:"-"`
+
+	PasswordSecretRef *v1.LocalSecretKeySelector `json:"passwordSecretRef,omitempty" tf:"-"`
+}
+
+type KeystoresAliasesKeyCertFileObservation struct {
+	Cert *string `json:"cert,omitempty" tf:"cert,omitempty"`
+
+	CertsInfo *CertsInfoObservation `json:"certsInfo,omitempty" tf:"certs_info,omitempty"`
+
+	Environment *string `json:"environment,omitempty" tf:"environment,omitempty"`
+
+	ID *string `json:"id,omitempty" tf:"id,omitempty"`
+
+	Keystore *string `json:"keystore,omitempty" tf:"keystore,omitempty"`
+
+	OrgID *string `json:"orgId,omitempty" tf:"org_id,omitempty"`
+
+	Type *string `json:"type,omitempty" tf:"type,omitempty"`
+}
+
+type KeystoresAliasesKeyCertFileParameters struct {
```

Comment:
Not a blocker but for better XRM compliancy: Similar concerns as above, with `OrgID`.
```